contrib/autorandr_launcher/autorandr_launcher.c

```diff
@@ -0,0 +1,165 @@
+#include <signal.h>
+#include <xcb/xcb.h>
+#include <xcb/randr.h>
+#include <getopt.h>
+#include <unistd.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <time.h>
+
+// indent -kr -i8
+static int VERBOSE = 0;
+extern char **environ;
+
+static void sigterm_handler(int signum)
+{
+	signal(signum, SIG_DFL);
+	kill(getpid(), signum);
+}
+
+static void ar_log(const char *format, ...)
+{
+	va_list args;
+
+	if (!VERBOSE)
+		return;
+	va_start(args, format);
+	vprintf(format, args);
+	va_end(args);
+	fflush(stdout);
+}
+
+static int ar_launch()
+{
+	pid_t pid = fork();
+	if (pid == 0) {
+		static char *argv[] =
+		    { "/usr/bin/autorandr", "--change", NULL };
```

Comment:
Please add `--default default` to make this behave the same way as the systemd service.

Reply:
Ok, I did it